iBridges.py

```diff
@@ -113,20 +115,21 @@ def setup_standard(self):
         """
         if self.standardButton.isChecked():
             self._init_envbox()
-            self.icommands = False
+            self.use_icommands = False
 
     def setup_icommands(self):
         """Check the state of the radio button for using iCommands.
-        This includes a check for the existance of the iCommands on the
+        This includes a check for the existence of the iCommands on the
         current system.
 
         """
         if self.selectIcommandsButton.isChecked():
             self.icommandsError.setText('')
-            if self.conn.icommands:
-                self.icommands = True
+            if self.conn.icommands.has_icommands:
+                self.use_icommands = True
                 # TODO support arbitrary iRODS environment file for iCommands
```

Comment:
The icommands do not work with an arbitrary irods_environment.json. It has to always be ~/.irods/irods_environment.json
So the hint for the GUI is: if the standard irods_environment.json is selected and the icommands are available, use the IrodsConnectorIcommands otherwise use the IrodsConnector.

Reply:
Near the bottom of https://docs.irods.org/4.3.0/system_overview/configuration/#irodsirods_environmentjson

> To use an environment file other than ~/.irods/irods_environment.json, set IRODS_ENVIRONMENT_FILE to load from a different location:
> 
> `export IRODS_ENVIRONMENT_FILE=/full/path/to/different.json`
> 
> Other individual environment variables can be set by using the UPPERCASE versions of the variables named above, for example:
> 
> `export IRODS_CWD='/tempZone/home/public'`
>

Reply:
I can add it to the CLI. So for the GUI, all is fine then.